src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/CommonTypes.Windows.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices.Tests
```

Comment:
Please put these 'Common' types files in a sub directory called `Common/` and update the namespace as appropriate.

Reply:
Sure, done